src/ckeditor.jsx

```diff
@@ -5,24 +5,67 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import EditorWatchdog from '@ckeditor/ckeditor5-watchdog/src/editorwatchdog';
+import uid from '@ckeditor/ckeditor5-utils/src/uid';
+import { ContextWatchdogContext } from './context.jsx';
+import ContextWatchdog from '@ckeditor/ckeditor5-watchdog/src/contextwatchdog';
```

Comment:
```suggestion
import EditorWatchdog from '@ckeditor/ckeditor5-watchdog/src/editorwatchdog';
import ContextWatchdog from '@ckeditor/ckeditor5-watchdog/src/contextwatchdog';
import uid from '@ckeditor/ckeditor5-utils/src/uid';
import { ContextWatchdogContext } from './context.jsx';
```

Reply:
I changed the order of imports. External packages (sorted A-Z), then local files.